mysql-test/suite/rocksdb/t/lock_wait_timeout_stats.test

```diff
@@ -0,0 +1,37 @@
+--source include/have_rocksdb.inc
+
+--enable_connect_log
+create table t (a int not null primary key) engine=rocksdb;
+
+begin;
+insert into t values (0);
+
+--source include/count_sessions.inc
+--connect (con1,localhost,root,,)
+--connection con1
+set @@rocksdb_lock_wait_timeout=1;
+select INNODB_ROW_LOCK_WAIT_TIMEOUTS from information_schema.table_statistics where table_name="t";
+begin;
+
+--connect (con2,localhost,root,,)
+--connection con2
+set @@rocksdb_lock_wait_timeout=1;
+begin;
+
+--connection con1
+--error ER_LOCK_WAIT_TIMEOUT
+insert into t values(0);
+select INNODB_ROW_LOCK_WAIT_TIMEOUTS from information_schema.table_statistics where table_name="t";
+select INNODB_ROW_LOCK_WAIT_TIMEOUTS from information_schema.table_statistics where table_name="t";
+
+--connection con2
+--error ER_LOCK_WAIT_TIMEOUT
+insert into t values(0);
+select INNODB_ROW_LOCK_WAIT_TIMEOUTS from information_schema.table_statistics where table_name="t";
+select INNODB_ROW_LOCK_WAIT_TIMEOUTS from information_schema.table_statistics where table_name="t";
+
+--disconnect con1
+--disconnect con2
```

Comment:
It seems like you shouldn't run disconnect on the connection you are currently on (from the mysqltest docs).